update.cf

```diff
@@ -68,7 +68,7 @@ bundle agent cfengine_internal_standalone_self_upgrade
         string => "$(sys.statedir)/MPF/desired-cfengine-package-version.json";
 
       "desired_pkg_data" -> { "ENT-3592" }
-        data => readjson( $(desired_pkg_data_path) ),
+        data => readjson( $(desired_pkg_data_path), inf ),
         if => fileexists( $(desired_pkg_data_path) );
```

Comment:
does "inf" translate to "infinity" somehow? I assume so but don't know where to look to check. Is it documented somewhere on docs.cfengine.com?

Reply:
yep. didn't scroll down enough. got it!